quarkchain/cluster/client_test.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import grpc
+import grpc_testing
+from grpc.framework.foundation import logging_pool
+
+import sys
+
+sys.path.append("quarkchain/cluster")
```

Comment:
is this needed?

Reply:
Otherwise it will not pass the circleci test. It will raise the error in that it could not import the module client.

Reply:
you probably want to import it as the whole path as 

```
import quarkchain.cluster.client
```